### ISSUE TITLE
Could you release the required label csv?

### ISSUE THREAD

**Issue by liuxiaotonglucy-a11y** [LXDxmu/GRMP-IQA#1]
I noticed that you used "image_labeled_by_score.csv" for pretraining. However, I didn't find it in your huggingface. Could you please release it for us to do experiments?

Thank you!

**Comment by ZzHowe** [comment]
Hello! Thank you for your interest in our work! You can find the missing file at [repo](https://github.com/zhuhancheng/MetaIQA/blob/master/LIVE_WILD/image_labeled_by_score.csv). This is because our pre-training section references the method from MetaIQA. We will also upload it to Hugging Face as a supplementary resource. Thank you again for your attention!

Zihao Huang
